### PR TITLE
Change from Overpass API to GooglePlaces API to fetch POIs

### PR DESCRIPTION
**Pull request by MarcelTorne**

### CODE REVIEW

File: app/src/main/java/com/github/epfl/meili/poi/PointOfInterest.kt [hunk 1]
```diff
@@ -9,22 +9,26 @@ data class PointOfInterest(
...
+        return START_CHAR + "POI:" + "lat:" + latitude + "long:" + longitude + ",name:" + name + ",uid:" + uid + ",icon:" + icon + END_CHAR
     }
 
     override fun equals(other: Any?): Boolean {
-        if (other != null && other::class.java == PointOfInterest::class.java){
-            val otherPoi =  other as PointOfInterest
-            return otherPoi.latitude.equals(latitude) && otherPoi.longitude==longitude && otherPoi.name == name && otherPoi.uid == uid
+        if (other != null && other::class.java == PointOfInterest::class.java) {
+            val otherPoi = other as PointOfInterest
+            return otherPoi.latitude.equals(latitude) && otherPoi.longitude == longitude && otherPoi.name == name
```
Comment (tberkane) [inline app/src/main/java/com/github/epfl/meili/poi/PointOfInterest.kt:24]: Nitpick: you can replace the call to equals by ==

**Review by tberkane — COMMENTED** [review]
Great job! Now that we have the custom POIs displayed on the map, maybe we should remove those from the google maps api?

**Comment by MarcelTorne** [comment]
> Great job! Now that we have the custom POIs displayed on the map, maybe we should remove those from the google maps api?

Yes I agree. I am going to ask on the team group to see if everyone is onboard with it. Thanks!

**Review by tberkane — APPROVED** [review]
Perfect!

**Review by aman3014 — APPROVED** [review]
Great work!

**Pull request merged**